src/Files.App/Data/Items/NavigationBarSuggestionItem.cs

```diff
@@ -1,17 +1,26 @@
 ﻿// Copyright (c) Files Community
 // Licensed under the MIT License.
 
+using Files.App.Controls;
+using Microsoft.UI.Xaml;
+
 namespace Files.App.Data.Items
 {
 	[Obsolete("Remove once Omnibar goes out of experimental.")]
-	public sealed partial class NavigationBarSuggestionItem : ObservableObject
+	public sealed partial class NavigationBarSuggestionItem : ObservableObject, IOmnibarTextMemberPathProvider
 	{
+		// Obsolete but left as is for backwards compatibility
+		private object? _Icon;
```

Comment:
This appears to be a new property, can it be removed?

Reply:
Removed.